src/test/app/forms/validation/validators/isCountrySupported.ts

```diff
@@ -0,0 +1,53 @@
+import { expect } from 'chai'
+import { CheckCountryConstraint } from 'forms/validation/validators/isCountrySupported'
+import { Country } from 'app/common/country'
+import { ValidationArguments } from 'class-validator'
+
+describe('IsCountrySupported', () => {
+  const constraint: CheckCountryConstraint = new CheckCountryConstraint()
+
+  describe('validate', () => {
```

Comment:
That tests should mock responses from the postcode lookup API to not use the real one that comes with an difficulty of possessing working API token. Please use the same approach as we have in route tests that uses 'nock' http server.